index.test.js

```diff
@@ -157,7 +161,7 @@ describe('handler', () => {
     expect(kinesis.__records[0]).toMatchObject({ type: 'bytes' })
   })
 
-  it.only('does not count segments until they are fully downloaded', async () => {
+  it('does not count segments until they are fully downloaded', async () => {
```

Comment:
Whoops, that was me!

And I'm guessing the `types`/`durations` at the top of the file were needed to make that first test actually pass?

Reply:
yes, I needed to add a little bit to it to make it pass